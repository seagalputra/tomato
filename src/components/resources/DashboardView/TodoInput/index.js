import React from 'react'

const TodoInput = () => {
  return (
    <form className="w-full p-2">
      <div className="block md:flex items-center">
        <input
          className="appearance-none w-full text-gray-700 mr-3 py-2 px-2 leading-light border border-teal-500 rounded"
          type="text"
          placeholder="To Do"
          aria-label="To Do"
        />
        <button
          className="flex-shrink-0 bg-teal-500 hover:bg-teal-700 border-teal-500 hover:border-teal-700 text-sm border-4 text-white py-2 px-2 rounded w-full md:w-32 mt-3 md:mt-0"
          type="button"
        >
          Add
        </button>
      </div>
    </form>
  )
}

export default TodoInput
