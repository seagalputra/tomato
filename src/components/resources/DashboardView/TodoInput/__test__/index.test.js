import React from 'react'
import { shallow } from 'enzyme'
import TodoInput from '..'

describe('TodoInput', () => {
  it('renders without crashing', () => {
    const component = shallow(<TodoInput />)
    expect(component).toMatchSnapshot()
  })
})
