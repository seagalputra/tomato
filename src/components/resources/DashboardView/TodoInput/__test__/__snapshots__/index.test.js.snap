// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoInput renders without crashing 1`] = `
<form
  className="w-full p-2"
>
  <div
    className="block md:flex items-center"
  >
    <input
      aria-label="To Do"
      className="appearance-none w-full text-gray-700 mr-3 py-2 px-2 leading-light border border-teal-500 rounded"
      placeholder="To Do"
      type="text"
    />
    <button
      className="flex-shrink-0 bg-teal-500 hover:bg-teal-700 border-teal-500 hover:border-teal-700 text-sm border-4 text-white py-2 px-2 rounded w-full md:w-32 mt-3 md:mt-0"
      type="button"
    >
      Add
    </button>
  </div>
</form>
`;
